hash/sega_beena_cart.xml

```diff
@@ -206,23 +456,24 @@ license:CC0
 		<description>Yes! PreCure 5 Asonde Oboeyou Hiragana!</description>
 		<year>2007</year>
 		<publisher>Bandai</publisher>
+		<info name="alt_title" value="Yes!プリキュア5 あそんでおぼえよう!ひらがな!"/>
```

Comment:
The two exclamation marks and the “5” in the Japanese part of the title should be fullwidth.

Reply:
Replaced all halfwidth with regular chars and removed spaces!